extensions/binding/org.eclipse.smarthome.binding.lifx/ESH-INF/thing/thing-types.xml

```diff
@@ -26,5 +27,12 @@
         <description>Selects the color of the light</description>
         <category>ColorLight</category>
     </channel-type>
+    
+    <channel-type id="temperature">
+        <item-type>Number</item-type>
```

Comment:
I realise that I have changed the code to accept I/D but not the type definition. will take care